test/libsolidity/SolidityABIJSON.cpp

```diff
@@ -46,7 +46,7 @@ class JSONInterfaceChecker
 
 		Json::Value generatedInterface = m_compilerStack.contractABI("");
 		Json::Value expectedInterface;
-		m_reader.parse(_expectedInterfaceString, expectedInterface);
+		BOOST_REQUIRE(m_reader.parse(_expectedInterfaceString, expectedInterface));
```

Comment:
I think this would throw if an error occurs so it should use `ETH_TEST_REQUIRE_NO_THROW`.